app/shared/favorite-button/FavoriteButtonContainer.spec.js

```diff
@@ -0,0 +1,55 @@
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import React from 'react';
+import simple from 'simple-mock';
+
+import { UnconnectedFavoriteButtonContainer } from './FavoriteButtonContainer';
+import FavoriteButton from './FavoriteButton';
+
+describe('shared/favorite-button/FavoriteButtonContainer', () => {
+  const resource = {
+    id: '123',
+    isFavorite: true,
+  };
+
+  const defaultProps = {
+    favoriteResource: simple.mock(),
+    unfavoriteResource: simple.mock(),
+    resource,
+  };
+
+  function getWrapper(props) {
+    return shallow(<UnconnectedFavoriteButtonContainer {...defaultProps} {...props} />);
+  }
+
+  it('renders a FavoriteButton', () => {
+    expect(getWrapper().is(FavoriteButton)).to.be.true;
+  });
+
+  it('has favorited prop', () => {
+    expect(getWrapper().prop('favorited')).to.be.true;
+  });
+
+  describe('handleClick', () => {
+    it('gets passed as onClick prop', () => {
```

Comment:
- [ ] This test should not be in this describe block. The "handleClick" describe block should only test the function.